blog/2022-12-14-dnsmasq-monitoring.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: dnsmasq-monitoring
+title: How to monitor and troubleshoot Dnsmasq DNS Forwarder
+description: Monitoring and troubleshooting Dnsmasq DNS Forwarder.
+authors: satya
+tags: [how-to, monitor, troubleshoot, dnsmasq,  dns, domain-name-server, DNS-management
+ dhcp, network-management]
+keywords: [how-to, monitor, troubleshoot, dnsmasq,  dns, domain-name-server, DNS-management
+ dhcp, network-management]
+image: https://user-images.githubusercontent.com/96257330/207541929-5857a8fe-393e-492f-bc24-410ca76f0d9b.png
+
+---
+
+Find out how to effectively and easily monitor and troubleshoot Dnsmasq DNS Forwarder using Netdata
+
+<!--truncate-->
+## What is Dnsmasq?
+
+Dnsmasq is an open-source, lightweight, DNS caching and forwarding server. It is designed to provide DNS resolution for small and home networks. Dnsmasq provides local DNS caching, forwarding, and recursive lookups, as well as DHCP, TFTP, and other related services. It also has support for DNS and DHCPv6, as well as various other features such as DNS-over-TLS and IPv6 privacy extensions. Dnsmasq is a versatile and highly configurable tool that is simple to
+We will cover Dnsmasq_DHCP in a separate blog with more details.
+
+## Monitoring Dnsmasq with Netdata
+
+The prerequisites for monitoring Dnsmasq with Netdata are to have DnsmasqDNSForwarder and [Netdata installed](https://learn.netdata.cloud/docs/cloud/get-started) on your system. 
+
+Netdata auto discovers hundreds of services, and for those it doesn't turning on manual discovery is a one line configuration. For more information on configuring Netdata for DnsmasqDNSForwarder monitoring please read the collector [documentation](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/dnsmasq).
+
+You should now see the DnsmasqDNSForwarder section on the Overview tab in Netdata Cloud already populated with charts about all the metrics you care about.
```

Comment:
Is DnsmasqDNSForwarder really the name of the section?

Reply:
Fixed. I missed this from the Autoblogger..
I did change the name in the rest of the blog. :)